contributor-guide/modules/ROOT/pages/testing/boost-test-matrix.adoc

```diff
@@ -1,286 +1,178 @@
-  --toolsets=TOOLSETS   comma-separated list of toolsets to test with
-  --libraries=LIBRARIES
-                        comma separated list of libraries to test
-  --incremental         do incremental run (do not remove previous binaries)
-  --timeout=TIMEOUT     specifies the timeout, in minutes, for a single test
-                        run/compilation
-  --bjam-options=BJAM\_OPTIONS
-                        options to pass to the regression test
-  --bjam-toolset=BJAM\_TOOLSET
-                        bootstrap toolset for 'bjam' executable
-  --pjl-toolset=PJL\_TOOLSET
-                        bootstrap toolset for 'process\_jam\_log' executable
-  --platform=PLATFORM   
-  --user=USER           Boost SVN user ID
-  --local=LOCAL         the name of the boost tarball
-  --force-update        do an SVN update (if applicable) instead of a clean
-                        checkout, even when performing a full run
-  --have-source         do neither a tarball download nor an SVN update; used
-                        primarily for testing script changes
-  --ftp=FTP             FTP URL to upload results to.
-  --proxy=PROXY         HTTP proxy server address and port
-                        (e.g.'http://www.someproxy.com:3128')
-  --ftp-proxy=FTP\_PROXY
-                        FTP proxy server (e.g. 'ftpproxy')
-  --dart-server=DART\_SERVER
-                        the dart server to send results to
-  --debug-level=DEBUG\_LEVEL
-                        debugging level; controls the amount of debugging
-                        output printed
-  --send-bjam-log       send full bjam log of the regression run
-  --mail=MAIL           email address to send run notification to
-  --smtp-login=SMTP\_LOGIN
-                        STMP server address/login information, in the
-                        following form: <user>:<password>@<host>[:<port>]
-  --skip-tests          do not run bjam; used for testing script changes
-
-```
-
-To test develop use "`--tag=develop`",
- and to test master use
- "`--tag=master`". Or substitute any Boost
- tree of your choice.
-
-
-### Details
-
-
-The regression run procedure will:
-
-
-* Download the most recent regression scripts.
-* Download the designated testing tool sources including
- Boost.Jam, B2, and the various regression
- programs.
-* Download the most recent from the [Boost Git Repository](/users/download/#repository)
- into the subdirectory boost.
-* Build b2 and process\_jam\_log if
- needed. (process\_jam\_log is a utility, which
- extracts the test results from the log file produced by B2).
-* Run regression tests, process and collect the
- results.
-* Upload the results to a common FTP server.
-
-
-The report merger process running continuously will merge
- all submitted test runs and publish them at [various locations](testing.html#RegressionTesting).
-
-
-### Advanced use
+* Git (a recent version).
+* At least 5 gigabytes of free disk space per compiler to be tested.
```

Comment:
Few of the tests below require 5GB of disk space. These requirements could be in the individual sections.